src/app/inventory/store/d1-item-factory.ts

```diff
@@ -189,7 +189,7 @@ function makeItem(
   }
 
   // def.bucketTypeHash is where it goes normally
-  let normalBucket = buckets.byHash[itemDef.bucketTypeHash];
+  let normalBucket = buckets ? buckets.byHash[itemDef.bucketTypeHash] : undefined;
```

Comment:
I don't know why or how buckets could ever be undefined here.

Reply:
https://sentry.io/organizations/destiny-item-manager/issues/1904169704/?project=279673

Reply:
Oh I know it happens. I don't know why or how it happens.

Reply:
totally misunderstood you, sorry